deno/wipe_appwrite_bucket/src/mod.ts

```diff
@@ -0,0 +1,47 @@
+import { sdk } from "./deps.ts";
+
+export default async function(req: any, res: any) {
+  if(
+    !req.variables['APPWRITE_FUNCTION_ENDPOINT'] || 
+    !req.variables['APPWRITE_FUNCTION_PROJECT_ID'] || 
+    !req.variables['APPWRITE_FUNCTION_API_KEY'] ||
+    !req.payload
+  ) {
+    return res.json({
+      success: false,
+      message: "Missing required environment variables or payload.",
+    });
+  }
+  let bucketId: string = '';
+  try {
+    const payload = JSON.parse(req.payload);
+    bucketId = payload.bucketId;
+  } catch(err) {
+    return res.json({
+      success: false,
+      message: "Payload is invalid.",
+    });
+  }
+  const client = new sdk.Client();
+  const storage = new sdk.Storage(client);
+  
+  client
+    .setEndpoint(req.variables['APPWRITE_FUNCTION_ENDPOINT'])
+    .setProject(req.variables['APPWRITE_FUNCTION_PROJECT_ID'])
+    .setKey(req.variables['APPWRITE_FUNCTION_API_KEY'])
+  ;
+
+  try {
+    for (const file of (await storage.listFiles(bucketId)).files) {
```

Comment:
By default, [`storage.listFiles()`](https://appwrite.io/docs/server/storage?sdk=nodejs-default#storageListFiles) returns the first 25 results. As such, this will only delete the 1st 25 files in the bucket. See our [Pagination](https://appwrite.io/docs/pagination) docs for more info.

Reply:
Nice catch, I thought that this will be enough for a demo. But I will fix it now :)